llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -1384,6 +1386,16 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
     }
   }
 
+  if (Subtarget.hasVendorXCValu()) {
```

Comment:
Can we pull the simple instructions out to a separate patch that can be easily approved?